src/views/Projects/index.jsx

```diff
@@ -0,0 +1,86 @@
+import { Component, Fragment } from 'react';
+import { withStyles } from 'material-ui/styles';
+import Grid from 'material-ui/Grid';
+import Button from 'material-ui/Button';
+import Typography from 'material-ui/Typography';
+import projects from '../../data/loader';
+import ProjectCard from '../../components/ProjectCard';
+
+@withStyles(
+  theme => ({
+    grid: {
+      padding: '0 15px !important',
+      maxWidth: '100%',
+    },
+    content: {
+      flexGrow: 1,
+      backgroundColor: theme.palette.background,
+      minWidth: 0,
+    },
+    container: {
+      paddingRight: '15px',
+      paddingLeft: '15px',
+      marginRight: 'auto',
+      marginLeft: 'auto',
+    },
+    flex1: {
+      flex: 1,
+    },
+    columnFlex: {
+      display: 'flex',
+      flexDirection: 'column',
+    },
+  }),
+  { withTheme: true }
+)
+export default class Projects extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      projects,
+    };
+  }
+  render() {
+    const { classes } = this.props;
+    const { projects } = this.state;
+
+    return (
+      <Fragment>
+        <header>
+          <div className={classes.columnFlex}>
+            <div className={classes.flex1} />
+            <Typography variant="display4" align="center">
+              Bug Issue
+            </Typography>
+            <Typography variant="display1" align="center">
+              A first contribution finder
+            </Typography>
+            <div className={classes.flex1} />
+          </div>
+        </header>
```

Comment:
Too many `div`s. Instead, we can do:

```jsx
<header className={classes.header}>
  <Typography variant="display1" align="center">
    Bug Issue
  </Typography>
  <Typography variant="subheading" align="center">
    Find your first contribution with Mozilla
  </Typography>
</header>
```

```js
header: {
  paddingTop: theme.spacing.unit,
  paddingBottom: 2 * theme.spacing.unit,
},
```

Note that I also changed the variant prop of both components to make sure we are consistent with our design.